source/Ocl/Converters/DefaultAttributeOclConverter.cs

```diff
@@ -35,7 +37,7 @@ public IEnumerable<IOclElement> ToElements(OclConversionContext context, string
                 if (!enumerable.GetEnumerator().MoveNext())
                     yield break;
 
-            yield return new OclAttribute(context.Namer.FormatName(name), obj);
+            yield return new OclAttribute(context.Namer.GetOclAttributeName(propertyInfo!), obj);
```

Comment:
could this be done in the calling code instead? That way the context wouldn't need to expose the namer in its internal state. It would also allow us to use the existing interface, and consumers could still override the name if desired.

Reply:
I'm not sure what you mean by the calling code?

Reply:
👀 ig nore me, I thought we had a different separation than we do. Turns out the "calling code" (by which I meant the thing calling `ToElements` and `ConvertInternal`) is just another converter, my bad.